code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -400,7 +400,7 @@ var/const/PATCH = 4 //patches
 
 /datum/reagents/proc/check_gofast(mob/M)
 	if(istype(M, /mob))
-		if(M.reagents.has_reagent("unholywater")||M.reagents.has_reagent("nuka_cola"))
+		if(M.reagents.has_reagent("unholywater")||M.reagents.has_reagent("nuka_cola")||M.reagents.has_reagent("stimulants"))

```

Comment:
could you please make these three into a single for loop? has_reagent shows up pretty high in the profile iirc, and combining these three into a single for loop would probably make it more efficient.

@RemieRichards is this a good idea code quality-wise though?